pkg/snapshot/snapshotter/snapshotter.go

```diff
@@ -108,6 +107,7 @@ type Snapshotter struct {
 	lastEventRevision    int64
 	K8sClientset         client.Client
 	snapstoreConfig      *brtypes.SnapstoreConfig
+	lastSecretModTime    time.Time
```

Comment:
There is no consistent convention in naming followed overall. In this case the field is named as `lastSecretModTime` whereas provider specific functions are named as `GetS3CredentialsLastModifiedTime` etc. Why can't we simply name this field as `lastSecretModifiedTime`?

Reply:
Understood, will make the necessary change.